modules/private-clusters-setting-api-private.adoc

```diff
@@ -43,16 +50,16 @@ lk4pj-worker-us-east-1b-zgpzg   running   m4.xlarge   us-east-1   us-east-1b   1
 +
 You modify the control plane machines, which contain `master` in the name, in the following step.
 
-. Remove the external load balancer from each control plane machine.
-.. Edit a control plane `Machine` object to remove the reference to the external load balancer.
+.. Remove the external load balancer from each control plane machine.
+... Edit a control plane `Machine` object to remove the reference to the external load balancer.
```

Comment:
```suggestion
... Edit a control plane `Machine` object to remove the reference to the external load balancer:
```
I know you didn't add this line, but would you mind fixing this while you're in here?